edx_name_affirmation/signals.py

```diff
@@ -0,0 +1,179 @@
+# pylint: disable=logging-format-interpolation
+
+""""
+Signal handlers for IDV and proctoring service signals
+"""
+
+import logging
+
+from django.contrib.auth import get_user_model
+
+from edx_name_affirmation.models import VerifiedName
+from edx_name_affirmation.statuses import VerifiedNameStatus
+from edx_name_affirmation.toggles import is_verified_name_enabled
+
+try:
+    from lms.djangoapps.verify_student.signals import idv_update_signal
+except ImportError:
+    idv_update_signal = None
+
+try:
+    from edx_proctoring.utils import exam_attempt_status_signal
```

Comment:
Same here, could you add comments here to explain why the import might not be successful?